src/ducks/settings/helpers.js

```diff
@@ -48,9 +48,8 @@ export const fetchSettings = async client => {
   return getDefaultedSettingsFromCollection(settingsCol)
 }
 
-export const updateSettings = async (client, newSettings) => {
-  const col = client.collection(DOCTYPE)
-  await col.update(newSettings)
+export const saveSettings = async (client, newSettings) => {
+  await client.save(newSettings)
```

Comment:
Nit: At this point, I think we can remove saveSettings, it complexifies the code for nothing.